bridges/snowbridge/pallets/inbound-queue/src/mock.rs

```diff
@@ -243,6 +249,38 @@ impl MessageProcessor for DummySuffix {
 	}
 }
 
+pub struct DeliveryCostReward<T>(sp_std::marker::PhantomData<T>);
```

Comment:
Instead of this, I would directly create in the pallet an implementation of the trait that fullfills the previous behavior, and use it in tests